timemachine/fe/free_energy.py

```diff
@@ -865,25 +902,28 @@ def log_q_fn(replica_idx: ReplicaIdx, state_idx: StateIdx) -> float:
         # Add an identity move to the mixture to ensure aperiodicity
         neighbor_pairs = [(StateIdx(0), StateIdx(0))] + neighbor_pairs
 
-    def get_equilibrated_initial_state(initial_state: InitialState) -> InitialState:
+    def get_equilibrated_xvb(xvb: CoordsVelBox, params: NDArray) -> CoordsVelBox:
         if md_params.n_eq_steps == 0:
-            return initial_state
+            return xvb
 
-        ctxt = get_context(initial_state)
+        context.set_x_t(xvb.coords)
+        context.set_v_t(xvb.velocities)
+        context.set_box(xvb.box)
 
-        xs, boxes = ctxt.multiple_steps(md_params.n_eq_steps, store_x_interval=0)
-        assert len(xs) == len(boxes) == 1
+        context.get_potentials()[0].set_params(params)
```

Comment:
Should we instead do `potential.unbound_potential.set_params`? Seems more explicit to access the single potential that we constructed than to access the 'first' potential on the context, or perhaps we should assert that the context only has one potential?

Reply:
Hmm, I think this is how it was before https://github.com/proteneer/timemachine/pull/1151/commits/78b89f7bf31bc3b9b913be2e48b6d537f0049a9f? I found it more explicit to access the potential via the context to make it clear that they are linked, but can sort of see your argument too. Can add an assertion that `len(context.get_potentials()) == 1`.

Reply:
Added assertion in https://github.com/proteneer/timemachine/pull/1151/commits/cae4a91c998abe2794f41698ecf698fa70a04f81